exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>"The <strong>flavor_customization</strong> build step consists of a Dockerfile and several package lists which can be modified. " -&gt; "The <strong>flavor_customization</strong> build step consists of modifying the Dockerfile and/or several package lists. "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Hm, but the build step itself does not consist of modifying the Dockerfile, nor the package lists.</p><p>What do you think about: : The <strong>flavor_customization </strong>build step is described by the Dockerfile and several package list files. We recommend to modify the package list files to add new packages to the flavor and only modify the Dockerfile if you need very specific changes, like adding additional resources.</p>

Reply:
<p>Fixed during a phone call together with @ahsimb</p>